services/notification/src/app.py

```diff
@@ -157,9 +161,11 @@ def effects_reactor_callback(world):
         **callback_metadata(properties(world)),
         'links': world.get('output', DEFAULT_OUTPUT),
     }
-    response = requests.post(url, data=data)
+    logger.info(f'Posting to {url} this data: {json.dumps(data)}')
```

Comment:
not sure we want to write the account connection info to the log...

Reply:
It's all data about the customer's account. It's definitely helpful for debugging and I thought the customer might like to see the data we're recording. Let me know if you are really think it should be removed.